packages/ckeditor5-ui/src/formheader/formheaderview.ts

```diff
@@ -61,41 +66,69 @@ export default class FormHeaderView extends View {
 	 */
 	constructor(
 		locale: Locale | undefined,
-		options: {
-			label?: string | null;
-			class?: string | null;
-			icon?: string | null;
-		} = {}
+		options: FormHeaderViewOptions = {}
 	) {
 		super( locale );
 
 		const bind = this.bindTemplate;
 
+		this.options = options;
+		this.iconView = this._createIcon();
```

Comment:
Is this always required? Maybe we could create it only when it is needed?

Reply:
In the function itself we check if an icon was passed, and if not, then it returns `null`.